features/bsm/rest/src/test/java/org/opennms/web/rest/v2/BusinessServiceRestServiceIT.java

```diff
@@ -190,6 +193,7 @@ public void canRetrieveBusinessServices() throws Exception {
         bsDTO.setName(bs.getName());
         bsDTO.setId(id);
         assertEquals(bsDTO, businessServices.get(0));
+        assertTrue("canRetrieveReductionKey", businessServices.get(0).getReductionKeys().contains(reductionKey));
```

Comment:
If you use assertTrue/assertFalse please add a meaningful description if the condition is not met, e.g.:
`assertTrue("It was expected that the given reductionkey " + reductionKey + " is contained in the reduction key set of the business service " + businessService.get(0).getName() + " but it was not", businessServices.get(0).getReductionKeys().contains(reductionKey))`